recipe/meta.yaml

```
@@ -18,40 +18,53 @@ build:
 
 requirements:
   host:
-    - python >=3.7
+    - python >=3.7 #,<3.8
```

```suggestion
    - python >=3.7
```
I don't think we need to include an upper bound for now